### DISCUSSION TITLE
I have make the payment but still cannot get the license.

### DISCUSSION THREAD

**Discussion by henrytiong** [aclap-dev/video-downloadhelper#336] [License & Purchase requests]
### What's your issue?

I have make the payment but still cannot get the license, my bank statement shown that the payment already made.

### The 10 first digits (only 10) of your license number or email address

henry.tiong1988@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GabrielHirel** [comment]
Good morning,
I have the same problem. 
My address hirel.gabriel@orange.fr

**Comment by jcfrog** [comment]
Hi,
I sent you emails.
jerome

**Comment by eduarduhh** [comment]
Good morning,
I have the same problem.
My address [eduarduhh@live.com](mailto:eduarduhh@live.com)
